cascades-server/configuration/src/main/java/pl/gov/coi/cascades/server/presentation/PresentationConfiguration.java

```diff
@@ -33,8 +36,13 @@ DatabaseNameGeneratorService produceDatabaseNameGeneratorService() {
     }
 
     @Bean
-    UsernameAndPasswordCredentialsGeneratorService produceCredentials() {
-        return new UsernameAndPasswordCredentialsGeneratorService();
+    UsernameAndPasswordCredentialsGeneratorService produceCredentials(Random random) {
+        return new UsernameAndPasswordCredentialsGeneratorService(random);
+    }
+
+    @Bean
+    Random produceRandomGenerator() {
```

Comment:
`Random` producer may be not needed as Spring Boot does this automatically, at least I think (need to verify).

If really needed it should be moved to some other Configuration class, more generic.